dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/cancel/CancelOrdersByAddressAndIdsSpec.scala

```diff
@@ -71,16 +108,43 @@ class CancelOrdersByAddressAndIdsSpec extends MatcherSuiteBase with ApiKeyHeader
 
     "should return an error when address is not a correct base58 string" in {
       validateMatcherError(
-        dex1.rawApi.cancelOrdersByIdsWithKey("null", placeAndGetIds(3)),
+        dex1.rawApi.cancelOrdersByIdsWithKeyOrSignature("null", placeAndGetIds(3)),
         StatusCode.BadRequest,
         InvalidAddress.code,
         "Provided address is not correct, reason: Unable to decode base58: requirement failed: Wrong char 'l' in Base58 string 'null'"
       )
     }
 
-    shouldReturnErrorWithoutApiKeyHeader(dex1.rawApi.cancelOneOrderWithKey(order.idStr(), headers = Map.empty))
```

Comment:
probably we should move this test to a proper place

Reply:
https://github.com/wavesplatform/matcher/blob/7f5429221128a0fe73785d3996f2395d55bfc945/dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/cancel/CancelOrderByIdSpec.scala#L97